feat(ip-ranges): Add Organization Domain IP Address. (#2515)
getsentry/sentry-docs: src/docs/product/security/ip-ranges.mdx

</markdown>

> </Alert>

Reviewer: might want to convert this to the new note format:

<Note>
<markdown>

This page applies only to Sentry's SaaS product. It **does not**
apply to Self-Hosted or Single Tenant.

</markdown>
</Note>

Author: @PeloWriter: now that I spent more time reading this, _what_ new format are you referring to?

## Dashboard and API

> Sentry's dashboard and API are served from a single IP address for all web traffic. The IP address is:

Reviewer: ```suggestion
Sentry's dashboard and API are both served from a single IP address for all web traffic. The IP address is:
```

```python
35.188.42.15/32
```

## Event Ingestion

> Sentry's Event Ingestion respects two domains within a Data Source Name (DSN): [Organization Subdomains](https://forum.sentry.io/t/organization-subdomains-in-dsns/9360) and Sentry's apex domain.

Reviewer: ```suggestion
Sentry's Event Ingestion respects two domains within a Data Source Name (DSN): 
1. The [organization subdomain](https://forum.sentry.io/t/organization-subdomains-in-dsns/9360) 
2. Sentry's apex domain
```

Sentry's primary domain accepts events from the same IP address as the Dashboard and API:

```python
35.188.42.15/32
```

> Starting $DATE, Sentry's Organization Subdomains will migrate from the prior IP address to the following single address:

Reviewer: ```suggestion
Starting $DATE, Sentry's organization subdomains will migrate from the prior IP address to the following single address:
```

Reviewer: Do we want to explain that there's no impact to existing customers? "This will not require any action for existing customers."

Author: The action required will be announced via blogpost/broadcast like [this one](https://blog.sentry.io/2017/06/15/notice-of-address-change).

Reviewer: 👍
